src/static/js/header_additional.js

```diff
@@ -223,7 +223,7 @@ additional = {
         // return additional.map((item) => ({ ...item, VALUE: (row.hasOwnProperty(item.FIELDNAME.toUpperCase()) ? row[item.FIELDNAME.toUpperCase()] : "") }));
         $.each(additional, function(i, a) {
             let fieldname = a.FIELDNAME.toUpperCase();
-            a.VALUE = "";
+            a.VALUE = a.DEFAULTVALUE;
```

Comment:
I don't think this is right here - it's going to set the default value when you're editing a citation if the additional field is blank. 

Instead, I think the citations screen should be calling additional.reset_defaults(selector) in the onload callback of dialog_show_add